pymc3/tests/test_distributions_random.py

```diff
@@ -498,6 +492,48 @@ class TestMoyal(BaseTestDistribution):
     ]
 
 
+class TestSkewNormal(BaseTestDistribution):
+    pymc_dist = pm.SkewNormal
+    pymc_dist_params = {"mu": 0.0, "sigma": 1.0, "alpha": 5.0}
+    expected_rv_op_params = {"mu": 0.0, "sigma": 1.0, "alpha": 5.0}
+    reference_dist_params = {"loc": 0.0, "scale": 1.0, "a": 5.0}
+    reference_dist = seeded_scipy_distribution_builder("skewnorm")
+    tests_to_run = [
+        "check_pymc_params_match_rv_op",
+        "check_pymc_draws_match_reference",
+        "check_rv_size",
+    ]
+
+
+class TestSkewNormalTau(BaseTestDistribution):
+    pymc_dist = pm.SkewNormal
+    pymc_dist_params = {"mu": 0.0, "tau": 1.0, "alpha": 5.0}
```

Comment:
Just realized sigma will be the same as tau (1). We should choose a test value where tau and sigma differ

Reply:
Indeed, I just noticed that. I might as well use the `get_tau_sigma` function.